parsers/to-css-textStyle/README.md

```diff
@@ -0,0 +1,88 @@
+    exclude?: Array<'color' | 'font-family' | 'font-size' | 'line-height' | 'letter-spacing' | 'text-align' | 'vertical-align' | 'text-transform' | 'font-variant' | 'text-decoration' | 'text-indent' | 'font' | 'fontSize' | 'lineHeight' | 'letterSpacing' | 'textAlign' | 'textTransform' | 'fontVariant' | 'textDecoration' | 'textIndent'>,
+    prefix?: string;
+    suffix?: string;
+    colorFormat?: 'rgb' | 'prgb' | 'hex' | 'hex6' | 'hex3' | 'hex4' | 'hex8' | 'name' | 'hsl' | 'hsv';
+    cssClassFormat?: 'camelCase' | 'kebabCase' | 'snakeCase';
+    fontFamilyFormat?: 'camelCase' | 'kebabCase' | 'snakeCase';
+    genericFamily?: 'serif' | 'sans-serif' | 'cursive' | 'fantasy' | 'monospace';
+    relativeLineHeight?: boolean;
+    prettierConfig?: Partial<{
+      endOfLine: 'auto' | 'lf' | 'crlf' | 'cr';
+      tabWidth: number;
+      useTabs: boolean;
+    }>;
+  }
+}
+```
+
+### Options
+| parameter | Require    | type      | default    | description                                       |
+| --------- | ---------- | --------- | ---------- | ------------------------------------------------- |
+| `include`    | optional   | `Array<string>`   |  | List of properties to include in css classes |
+| `exclude`    | optional   | `Array<string>`   |  | List of properties to exclude in css classes |
+| `prefix`    | optional   | `string`   |  | A string will be append before the css class name |
+| `suffix`    | optional   | `string`   |  | A string will be append after the css class name |
+| `colorFormat`    | optional   | `rgb, prgb, hex, hex6, hex3, hex4, hex8, name, hsl, hsv`   | `rgb` | A color format applied when a text style include a color |
+| `cssClassFormat`    | optional   | `camelCase, kebabCase, snakeCase`   | `kebabCase` | The lodash function used to normalize the css class name |
+| `fontFamilyFormat`    | optional   | `camelCase, kebabCase, snakeCase`   |  | The lodash function used to normalize the font family value |
+| `genericFamily`    | optional   | `string` |  | The generic font family will be applied after the main font family |
+| `relativeLineHeight`    | optional   | `boolean` | | Convert line height to relative value |
+| `prettierConfig.endOfLine`    | optional   | `auto, lf, crlf, cr`   | `auto` | [Prettier documentation](https://prettier.io/docs/en/options.html#end-of-line) |
+| `prettierConfig.tabWidth`    | optional   | `number`   | `2` | [Prettier documentation](https://prettier.io/docs/en/options.html#tab-width) |
+| `prettierConfig.useTabs`    | optional   | `boolean`   | `false` | [Prettier documentation](https://prettier.io/docs/en/options.html#tabs) |
+
+## Usage example
+
+```json
+{
+    "name": "to-css-text-style",
+    "options": {
+      "exclude": ["color", "text-indent"],
+      "prefix": "sp-",
```

Comment:
Instead of "sp" which doesn't mean anything for someone outside Specify, I'd rather be more explicit by having the following value:
- "prefix": "yourPrefix"